homeassistant/components/device_tracker/freebox.py

```diff
@@ -0,0 +1,144 @@
+"""
+Support for device tracking through Freebox routers.
+
+This tracker keeps track of the devices connected to the configured Freebox.
```

Comment:
Move the rest of the docstring from this line to the documentation. Replace this line with a link to the documentation. See other platforms for the correct link format.